doc/manual/en/config/widgets/plugins/tr064/index.rst

```diff
@@ -151,18 +151,33 @@ The user itself needs the rights to access the call list:
 
 .. figure:: _static/fritzbox_user.png
 
-That the plugin can access this user account the credentials must be given.
-This can be done in the CometVisu manager in the area "hidden configuration"[#f2]_:
+For the plugin to be able to use this user its credentials must be given. This
```

Comment:
This sentence just does not "sound right". Maybe: "The plugins needs to know the credentials to be able to use this user." would be better.